### ISSUE TITLE
Rails 5 Compatibility 

### ISSUE THREAD

**Issue by nicwillemse** [syedmusamah/grape_on_rails_routes#5]
According to the gemspec this will not work on rails 5 ? Any plans to change this ?

**Comment by syedmusamah** [comment]
@nicwillemse , just need to test compatibility with rails 5 and update the rails version requirement. 

I can get to this in the coming week or so, but feel free to make a PR ;)

**Comment by nbulaj** [comment]
[Rails 5 is released](http://guides.rubyonrails.org/5_0_release_notes.html), so we need to support it now.

```
grape_on_rails_routes was resolved to 0.3.0, which depends on
      rails (< 5, >= 3.1.1)

```

**Comment by syedmusamah** [comment]
@nicwillemse and @nbulaj , I just published a new version!
